pdp/policy.go

```diff
@@ -59,8 +59,16 @@ func NewPolicy(ID string, hidden bool, target Target, rules []*Rule, makeRCA Rul
 }
 
 func (p *Policy) describe() string {
+	ruleIDs := make([]string, len(p.rules))
+	ruleIdx := 0
+	for _, rule := range p.rules {
+		if ruleID, ok := rule.GetID(); ok {
+			ruleIDs[ruleIdx] = ruleID
+			ruleIdx++
+		}
+	}
```

Comment:
Is it worthwhile adding a limit to the number of rules to display? Can policies have a lot of rules?

Reply:
Yes, they can. And yes you have to.